package.json

```diff
@@ -12,8 +12,7 @@
   },
   "scripts": {
     "pretest": "npm run build",
-    "test": "jest",
-    "test:coveralls": "npm run build && jest --coverage && cat ./coverage/lcov.info | coveralls",
```

Comment:
Looks like Coveralls isn't hooked up here either, so I removed the dependency and the separate `test:coveralls` script. Jest will now always run with coverage reporting enabled.